package.json

```diff
@@ -13,23 +13,23 @@
 		"prepack": "npm run test && npm run build"
 	},
 	"peerDependencies": {
-		"vue": "2.5.17"
+		"vue": "~2.6.10"
```

Comment:
Is this ~ intended?

Reply:
Yes. We should not force the consumer of this module to have exactly Vue 2.6.10. Any 2.6 should be fine.